### DISCUSSION TITLE
Cloud Firestore issue while building Flutter IOS App in Code Magic

### DISCUSSION THREAD

**Discussion by nischal-subedi** [firebase/flutterfire#7539] [Q&A]
Hi Guys!,

I've been trying to build an Flutter Application for IOS since the past week on Code Magic. The Application successfully builds for Android as well as Web but the IOS part of the build keeps on failing.

Everytime i run the build I get an XCODE error message that says Cloud Firestore was not imported.

`Xcode's output:
↳
    /Users/builder/clone/ios/Runner/GeneratedPluginRegistrant.m:12:9: fatal error: module 'cloud_firestore' not found
    @import cloud_firestore;
     ~~~~~~~^~~~~~~~~~~~~~~
    1 error generated.`

I've connected the Apple Developer Account to Code Magic so there's no issue with App Signing, for context I was facing this error before but once i connected the Developer Account it was fixed. Now the only thing remaining is cloud_firestore and I have no clue what is causing the error. All solutions mentioned on Github as well as Stackoverflow have been tried but I haven't been able to fix the solution.

Please help me!!

**Comment by BetoMacias** [comment]
Hi @nischal-subedi 

Did you find the fix for this issue?
